src/why3prover.ml

```diff
@@ -0,0 +1,97 @@
+(** Calling provers in Why3 *)
+
+open Timed
+
```

Comment:
Many spaces/lines on lines 5-19. Could be minimized.

Reply:
Changed in 1c845767eab3190fb6ba22b214e96664254246de